internal/config/validation.go

```diff
@@ -712,7 +712,7 @@ func returnsTrue(_ validator.FieldLevel) bool {
 }
 
 func registerValidateNameError(ut ut.Translator) error {
-	return ut.Add("validate_name", "{0} must be no more than {1} characters long", true)
+	return ut.Add("validate_name", "{0} must be no more than {1} characters long / contain only lowercase letters on AWS", true)
```

Comment:
```suggestion
	return ut.Add("validate_name", "{0} must be no more than {1} characters long and contain only lowercase letters", true)
```

The message will only appear when using AWS, so we don't need to mention this?

Reply:
I don't understand. If on e.g. GCP I set a name longer than 37 characters the error would be 
`name must be no more than 37 characters long and contain only lowercase letters` which is wrong